recommenders/evaluation/python_evaluation.py

```diff
@@ -736,6 +736,19 @@ def get_top_k_items(
 }
 
 
+# get col_rating column
+def _get_col_rating_column(relevancy_method:str, **kwargs) -> str:
```

Comment:
Please note that we use Google doc format for the docstring, can you please update?

Reply:
Thanks for the feedback, I have updated the same. LMK if it looks good now!